docs/process.md

```diff
@@ -41,105 +42,65 @@ Costs are
+
+You are taking the temperature of the list.  If there is a great outcry that this is a bad idea, or has been tried before, or was explicitly rejected before, you should probably stop.  If everyone says, "Sounds great, flesh it out!", or if you're not sure, you should post, requesting that the PSC approve producing a draft.  If they do, move on to "Draft Proposal" below.  If not, they'll provide more feedback like "this is definitely impossible" or "you need to provide more information" or so on.
+
+During this "Pre-RFC" phase, your proposal isn't in the RFC tracker.  It's not an RFC yet!
+
+During this phase, the proposer is responsible for moving things forward.  If
+they contact the PSC for approval to file a draft, and the PSC does not
+respond, it's the proposer who should be keeping track of that.
+
+### Draft Proposal
+
+The PSC has agreed that you should write a formal draft proposal.  You get the template document and fill it out.  You take its advice, thinking hard about what goes in each section.  Then you post it to p5p as an email with the subject "PROPOSAL:  my great idea".  Members of the list will reply with more questions and suggested amendments.  You should read them and amend the proposal to clarify your ideas or react to valid criticism.
+
+During this phase, you (the proposer) are responsible for moving things
+forward.
+
+When you think your idea has been sufficiently scrutinized, and you gotten all the feedback you're going to benefit from, submit your updated document to the PSC.  They will either request more discussion, reject the proposal, or enter it into the tracker with the status **Exploratory**.
+
+### Exploratory Status
+
+When a formal draft has been discussed sufficiently, submitted to the PSC, and is under consideration, it is entered into the proposal tracker, with the status **Exploratory**.  The PSC (or other deputized folk) will begin the process of vetting the idea.  They will review discussion about the idea, they will produce a list of questions not yet answered, and they will press existing core team members (or other experts) for input.
+
+The PSC (or deputies) will eventually either:
+ * move the document to **Implementing** status (see below) because they believe it is ready for implementation
+ * move the document to **Rejected** status because they believe it should not be implemented; This may come with advice on how to formulate an alternate proposal that has more chance of being accepted.  This isn't used for "it needs some edits", but for "it's fundamentally deeply flawed."
+ * move the document to the **Expired** status because the original proposer (or other vital personnel) are not responsive
+
+During this phase, the PSC is responsible for moving the proposal forward.
```

Comment:
I understand this is volunteer work (and I truly appreciate it), I just wonder if there is any sort of timeline/timeframe to be considered on the process itself (e.g. "RFCs in the exploratory status _should_ be discussed by the PSC at most in 3 subsequent PSC meetings, after which either a new status will be set, or the PSC will come forward with a list of questions. If said questions are not satisfactorily answered by the next 3 subsequent PSC meetings, it will move to either rejected or expired")

Reply:
I'm not opposed, but mostly I think that a big goal for me here is to make it clear whose responsibility it is, and setting a time frame is redundant and possibly bad.  All "responsible for moving forward" should carry with it "as rapidly as is practical and responsible."

Reply:
Agreed!